scripts/clean-old-namespace-from-cluster.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env sh
+# this scripts logins as kubeadmin and removes the namespaces that are more than oneday old.
+# this script is used in a kubernetes cronjob for PSI/IBM cloud openshift cluster.
+
+# login as kubeadmin
+if [[ $CLUSTER_TYPE == "PSI" ]]; then
+    ￼ #PSI cluster login
+    ￼ oc login -u kubeadmin -p ${OCP4X_KUBEADMIN_PASSWORD} --insecure-skip-tls-verify ${OCP4X_API_URL}
+else
+    ￼ # Login to cluster in IBM Cloud using cluster API key
+    ￼ oc login -u ${IBM_OC_LOGIN_USER} -p ${IBMC_ADMIN_OCLOGIN_APIKEY} --server=${IBMC_OCP47_SERVER}
+fi
+
+# PROJECT_AND_TIME var will contain namespace and date with time seperated with `|`
+# PROJECT_AND_TIME is selected using labels app and team. e.g: app=test and team=odo
+# eg. cmd-push-test157kgb|2021-08-17T12:40:20Z
+PROJECT_AND_TIME=$(kubectl get namespace -o jsonpath='{range .items[*]}{.metadata.name}{"|"}{.metadata.creationTimestamp} {"\n"}{end}' | grep -v '^openshift\|^kube\|^default\|^ibm\|^calico\|^tigera\|^odo-operator-test')
+
+for PROJECT in ${PROJECT_AND_TIME}; do
+    IFS='|' read -r PRJ TIME <<<"$PROJECT" # seperate the Namespace and time of creation using IFS(Input Field Seperators) value `|`
+
+    CONFIGMAP=$(kubectl get configmaps -n $PRJ -o jsonpath='{range .items[*]}{.metadata.name}{"\n"}' | grep config-map-for-cleanup) # check if PROJECT contains configmap named 'config-map-for-cleanup'
+                                                                                                                                    # if not then the value of CONFIGMAP will be empty
+    if [[ ! -z $CONFIGMAP ]]; then
+
+        TEAMANDTYPE=$(kubectl get configmaps $CONFIGMAP -n $PRJ -o jsonpath='{.data.team}{"|"}{.data.type}') # fetch team and tyoe(testing) data from the configmap
+        IFS='|' read -r TEAM TYPE <<<"$TEAMANDTYPE"                                                          # seperate the TEAM and TYPE of creation using IFS(Input Field Seperators) value `|`
+
+        if [[ $TYPE -eq "testing" ]]; then # check if type if testing
```

Comment:
I think we also want to filter on the team value

Reply:
understood, adding it ASAP